packages/cli/src/cli/cmd/i18n.ts

```diff
@@ -130,12 +133,73 @@ export default new Command()
             await bucketLoader.init();
 
             const sourceData = await bucketLoader.pull(i18nConfig!.locale.source);
-            lockfileHelper.registerSourceData(bucketPath.pathPattern, sourceData);
+
+            const deltaProcessor = createDeltaProcessor(bucketPath.pathPattern);
+            const checksums = await deltaProcessor.createChecksums(sourceData);
+            await deltaProcessor.saveChecksums(checksums);
           }
         }
-        ora.succeed("i18n.lock created");
+        ora.succeed("Localization cache initialized");
       } else {
-        ora.succeed("i18n.lock loaded");
+        ora.succeed("Localization cache loaded");
+      }
+      // Handle json key renames
```

Comment:
This is automatic on running `i18n`, right?

I can see 2 potential issues:

1. What if this is not actually a rename, but rather the same string was removed and added in different place? Eg. it could be translated differently because it is in different context.
2. I see the MD5 hash is calculated for all source file keys (to calculate `updated` array in delta processor), this might be slow for projects with 100s of files.

Shall we put this behind a flag (eg. `--migrate`) to run only when explicitly required?